### ISSUE TITLE
core/scripts: remove stuff from cpp that doesn't need to be there

### ISSUE THREAD

**Issue by mysterious-static** [mysterious-static/TrinityCore#5]
looking through zone_shadowmoon_valley.cpp (although I'm confident other scripts have this as well), stuff like npc_drake_dealer_hurlunk or npc_karynaku could, I believe, be ported into sai easily. this issue is just to remind me to talk to devs about it and make sure that it makes sense to do before I start doing stuff that I shouldn't be doing

**Comment by mysterious-static** [comment]
after discussing, I think for now it'll be on a case by case basis as I come across them. if I can migrate something, I will.

After #2 is done I'll go through every script and move what I can

**Comment by ghost** [comment]
That looks like a very good plan. :+1: Enjoy your work on it and don't hesitate to ask if you are uncertain.

**Comment by Aokromes** [comment]
You can close this now no? :P

**Comment by ghost** [comment]
I would think it depends on the ambitions expressed in the first post (although not very precise).

---

edit: looking through PR #17373 again, it does seem to cover this issue pretty well. I'll close it (if you find any reasons it should stay open, it can easily be reopened).

**Comment by mysterious-static** [comment]
oh okay yeah sorry this wasn't clear, this is kind of just a reminder to me to go through and do more than just gossips, or at least see what else I can do, in every script as I get to them

also to go through the ones that are already done later and review those too
